responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html

```diff
@@ -92,7 +92,7 @@
       <div class="row">
         <div class="col-12">
           <div class="review-panel data-panel">
-            <label class="tab-subtitle">Household Members</label>
+            <label class="tab-subtitle">Household Members & Pets</label>
```

Comment:
I recommend to keep the review page unchanged - only the edit pets should send the user to the household member and pets page

Reply:
I returned the previous state ( except the changes we have to adjust based on 2 questions being removed). And there is no link to the step from the review file